testcafe/pages/ServicePage.js

```diff
@@ -1,16 +1,23 @@
 import { ReactSelector } from 'testcafe-react-selectors';
 import config from '../config';
 
-export default class ResourcePage {
+export default class ServicePage {
   constructor() {
     const baseSelectorName = 'ServicePage';
     const baseSelector = ReactSelector(baseSelectorName);
-    this.serviceName = baseSelector.find('header h1');
-    this.description = baseSelector.find('.listing--main--left--about');
+    this.address = baseSelector.find('.listing--main--left--hours header strong');
+    this.appProcess = baseSelector.find('.listing--main--left--details tr').nth(0).find('td');
+    this.cost = baseSelector.find('.listing--main--left--details tr').nth(2).find('td');
+    this.description = baseSelector.find('.listing--main--left--about div');
     this.details = baseSelector.find('.listing--main--left--details');
+    this.directionsButton = baseSelector.find('.action-sidebar--directions');
     this.editButton = baseSelector.find('.action-sidebar--edit');
+    this.email = baseSelector.find('.listing--main--left--contact tr').nth(1).find('td');
+    this.name = baseSelector.find('.listing--main--left > header h1');
+    this.note = this.details.find('tbody td');
+    this.phone = baseSelector.find('.listing--main--left--contact tbody td');
     this.printButton = baseSelector.find('.action-sidebar--print');
-    this.directionsButton = baseSelector.find('.action-sidebar--directions');
+    this.requiredDocs = baseSelector.find('.listing--main--left--details tr').nth(1).find('td');
```

Comment:
`directionsButton` was moved above to alphabetization.  The diff this is confusing to read.